examples/Bickley_jet_shallow_water.jl

```diff
@@ -76,43 +69,51 @@ model = ShallowWaterModel(
 
 # ## Background state and perturbation
 #
-# The background velocity ``\overline{u}`` and free-surface ``\overline{\eta}`` are chosen to 
-# represent a geostrophically balanced Bickely jet with maximum speed of ``U`` and 
-# maximum free-surface deformation of ``Δη``, i.e.,
+# The background velocity ``ū`` and free-surface ``η̄`` are chosen to represent a 
+# geostrophically balanced Bickely jet with maximum speed of ``U`` and maximum 
+# free-surface deformation of ``Δη``, i.e.,
 #
 # ```math
-# \overline{\eta}(y) = - Δη \tanh(y) , \\
-# \overline{u}(y) = U \mathrm{sech}^2(y) .
+# \begin{aligned}
+# η̄(y) & = - Δη \tanh(y) , \\
+# ū(y) & = U \mathrm{sech}^2(y) .
+# \end{aligned}
 # ```
-# 
-# The vorticity of the background state is ``Ω = - \partial_y \overline{u} = 2 U \mathrm{sech}^2(y) \tanh(y)``.
 #
-# Linear stability theory predicts that for the particular parameters that we consider here,
-# the growth rate for the maximum growth rate should be ``0.14``.
+# The total height of the fluid is ``h = L_z + \eta``. Linear stability theory predicts that 
+# for the parameters we consider here, the growth rate for the most unstable mode that fits 
+# our domain is approximately ``0.14``.
 # 
-# We specify `Ω` to be the background vorticity of the jet in the absence of any perturbations.
-# The initial conditions include a small-ampitude perturbation that decays away from the center
-# of the jet.
-
-  Ω(x, y, z) = 2 * U * sech(y)^2 * tanh(y)
- uⁱ(x, y, z) =   U * sech(y)^2 + ϵ * exp(- y^2 ) * randn()
- ηⁱ(x, y, z) = -Δη * tanh(y)
- hⁱ(x, y, z) = model.grid.Lz + ηⁱ(x, y, z)
+# We also specify `ω̄` as the vorticity of the background state, 
```

Comment:
Again, overline looks weird but does not matter here as it does in the final product